### DISCUSSION TITLE
Whisper 3

### DISCUSSION THREAD

**Discussion by omiddjoudi** [deepgram/community#992] [Feature Requests]
Hello,
Is there a plan to have Whisper 3 in Whisper Cloud ?
Thanks

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- The deepgram product you are using (e.g Speech to Text, Agent API)
- A request ID that triggered your error or issue.

**Comment by deepgram-community** [comment]
Hi there! There is currently no plan to upgrade Whisper to Whisper 3 on our infrastructure. I'd love to know what it is that our Nova architecture models are unable to achieve that you feel Whisper 3 can

> ![avatar](https://avatars.slack-edge.com/2024-03-28/6876915321187_66505c29ed0cfe659927_32.jpg) This message was sent by luke from Deepgram, via our community automation.

**Reply by omiddjoudi** [reply]
Hi Luke, 
Thanks for your reply. We made some tests on a speech in french language from a conf room with noise and people talking same time :) and the result of transcription with Whisper was much better than Nova-2. 
But the diarizer is stellar, that's why we would like to use Deepgram :)

**Reply by tills3** [reply]
Dito, we have seen great results with Whisper 3 and would like to use it via Deepgram because we appreciate the diarizer as well! And we are curious for Nova-3, is there any update on when that might be released?
